docs/spark-standalone.md

```diff
@@ -240,6 +240,16 @@ SPARK_MASTER_OPTS supports the following system properties:
   </td>
   <td>1.6.3</td>
 </tr>
+<tr>
+  <td><code>spark.submit.waitAppCompletion</code></td>
```

Comment:
I think this is the wrong place; this shows properties you can set on the master, but this is an app setting. Right? or it should be?

Reply:
That's right, it's an app setting. I couldn't find a section for application settings in spark-standalone.md. Could you please suggest where we can add it? https://spark.apache.org/docs/latest/spark-standalone.html

Reply:
Or should it be added in the generic configuration application-properties section here: https://spark.apache.org/docs/latest/configuration.html#application-properties?

Reply:
Hm, maybe configuration.md, but it's specific to standalone.
I suppose you could mark the existing configs as 'cluster configs' and make a new table of 'client configs'?

Reply:
![image](https://user-images.githubusercontent.com/31816865/80607734-78b0e900-8a53-11ea-8275-e6a302c50539.png)
How about updating in this Application Properties section and making the config name as spark.submit.standalone.waitAppCompletion as suggested by @HeartSaVioR . Asking this since changing existing configs to cluster config may confuse w.r.t. existing documentation and general configuration has specific mode properties as well. Please let me know what you think. Thanks.

Reply:
Hm, there's not a great solution. I think I would add a new section in the standalone docs page, after "Connecting an Application to the Cluster", called "Spark Properties", and make a table with this single property. I'd call it spark.standalone.submit.waitAppCompletion" for full consistency with YARN et al.

Reply:
Sounds good, will do that.

Reply:
Updated the documentation file, please review.